processor/src/test/java/org/mapstruct/ap/test/bugs/_1830/BeanMapperWithStrategyDefinedInConfig.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._1830;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.NullValuePropertyMappingStrategy;
+
+@Mapper(nullValuePropertyMappingStrategy = NullValuePropertyMappingStrategy.CLEAR)
+public interface BeanMapperWithStrategyDefinedInConfig {
```

Comment:
The name of the mapper does not reflect what it is doing. The strategy is defined in the mapper not in a config. A config is a `@MapperConfig`